perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -495,9 +495,27 @@ sub look_up_property {
 
     my $attributes = $self->property_attributes($uprn);
     $premises{$uprn}{attributes} = $attributes;
+
+    $premises{$uprn}{has_pending_bulky_collection}
+        = $self->has_pending_bulky_collection( $premises{$uprn} );
+
     return $premises{$uprn};
 }
 
+sub has_pending_bulky_collection {
+    my ( $self, $property ) = @_;
+
+    my @collections = FixMyStreet::DB->resultset('Problem')->search(
+        {   category => 'Bulky collection',
+            detail   => { -like => '%' . $property->{address} . '%' },
```

Comment:
This works well right now, but I'd be worried about it breaking if we did something like stopped storing the address in the detail field, or changed the way addresses are formatted, etc etc. Looking for a more stable value like the UPRN in the `extra` is a better bet. Something like this:
```suggestion
            extra => { like => '%T7:uprn,T' . length($property->{uprn}) . ':' . $property->{uprn} . '%'},
```

(aside, yes, it's currently a bit horrid to query the `extra` field. There is [a PR](https://github.com/mysociety/fixmystreet/pull/3220) that'll make it much easier)